app/views/layouts/admin/_page.html.slim

```diff
@@ -1,6 +1,13 @@
 - content_for :styles
   = stylesheet_link_tag "admin/content", :media => "all"
 
+- content_for :scripts
+
+  javascript:
```

Comment:
Sure, and yeah, I was just copying what I saw in this project..  Should the 'content_for :styles' block also go at the bottom?